email-notifications.js

```diff
@@ -15,17 +16,26 @@ module.exports = function (options) {
   seneca.add({role: plugin, cmd: 'send'}, send_notification);
 
   function send_notification (args, done) {
+    var subject = args.subject;
+    var subjectVariables = args.subjectVariables || [];
+    var subjectTranslation;
     if (options.sendemail && options.email) {
       var emailCode = args.code + args.locality;
       if (!fs.existsSync(path.join(__dirname, '/email-templates/', emailCode))) emailCode = args.code + 'en_US';
       if (!args.to) return done(null, {ok: false, why: 'No recipient set.'});
+
+      subjectTranslation = i18nHelper.getClosestTranslation(args.locality, subject);
```

Comment:
This will try get an exact match for the given locality, and if it doesn't find it, it'll get the en_US value. This should always exist regardless.